api/filters/logging.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package filters
```

Comment:
I am not sure how useful switching logging level per request is. The idea is that normally we get a ticket open because a certain existing flow is failing (e.x. a call from a component calling sm is failing) - it will not be ideal (and probably not always possible for us (sm operators) to imperosanate the exact flow/ call that is failing and attach a request-log header.

It would make sense if we had an API secured with admin scope that would allow to dinamically switch the log level used - either in general (in case we decide we don't want to store loggers and we create them always on each request) or for particular packages (if we decide that we want to store the loggers so that they can be reused and overhead of creating them each time is avoided)

Reply:
When using the CLI you might set it in debug mode which also sends `X-Log-Level: DEBUG` in all requests. This would be useful IMO.

I agree it would be useful to make this management API. Do you agree to open an issue and follow it with a separate PR?

Reply:
https://github.com/Peripli/service-manager/issues/132